spec/files/CRLF/Sniffs/Files/LineEndingsSniff.php

```diff
@@ -0,0 +1,133 @@
+<?php
```

Comment:
Isn't this a sniff that is included with PHPCS itself? Is there a reason you are embedding the file here?

Reply:
The sniff that is included with PHPCS only puts the error on the first char of the line. This one puts an error at the end of every line.

Reply:
Do you want me to trim the file down? This was the sniff that I noticed the error on.